python/langsmith/async_client.py

```diff
@@ -0,0 +1,714 @@
+"""The Async LangSmith Client."""
+
+from __future__ import annotations
+
+import asyncio
+import datetime
+import logging
+import uuid
+import warnings
+from typing import (
+    Any,
+    AsyncIterator,
+    Dict,
+    List,
+    Mapping,
+    Optional,
+    Sequence,
+    Tuple,
+    Union,
+    cast,
+)
+
+import httpx
+
+from langsmith import client as ls_client
+from langsmith import schemas as ls_schemas
+from langsmith import utils as ls_utils
+from langsmith._internal import _beta_decorator as ls_beta
+
+logger = logging.getLogger(__name__)
```

Comment:
nit: should we use an async logger here? We do that with `structlog` on langsmith

Reply:
TIL. I actually am not using this anymore so will just remove. Think async logger would add another pckg right?